acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -193,6 +193,23 @@ public void createYubiHsmYamlFileAt(final Path metadataFilePath, final KeyType k
     createYamlFile(metadataFilePath, yaml);
   }
 
+  public void createInterlockYamlFileAt(
```

Comment:
Interlock _can_ be run simply as an app on a PC ... so thinking we should refer to this HSM style as Interlock, rather than an Armory HSM 👍